src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=1853644024&format=interactive",
+  sheets_gid="1811313356",
+  sql_file="durable_storage_estimate_usage.sql"
+  )
+}}
+
+The Storage Manager API is supported in Chrome since 2016, Firefox since 2017, and the new Chromium-based Edge. HTTP Archive data shows that the API is used in 0.49 percent of all desktop websites tracked by the archive, and 0.48 percent of all mobile websites. Over the course of 2020, the usage of the Storage Manager API kept growing. This also makes this interface the most commonly used API in this chapter.
+
+### Opt-in to persistent storage
+
+There are two categories of web storage: "Best Effort" and "Persistent", with the first being the default. When a device is low on storage, the browser automatically tries to free up best effort storage. For instance, Firefox and Chromium-based browsers evict storage from the least recently used origins. This, however, poses a risk of losing critical data. To prevent eviction, developers can opt for persistent storage. In this case, the browser will not clear the storage, even when space is low. Users can still choose to clear up the storage manually. To opt for persistent storage, developers need to call the `navigator.storage.persist()` method. Depending on the browser and site engagement, a permission prompt may show, or the request will automatically be accepted or denied.
+
+{{ figure_markup(
+  image="storage_manager_api_persist.png",
+  alt="Percent of top pages using the persist method of the StorageManager API",
+  caption="Percent of top pages using the persist method of the StorageManager API.",
+  description="Chart of the usage of StorageManager API's persist method, based on the number of pages monitored by HTTPArchive. It compares the usage on mobile and desktop devices. On desktop pages, the usage is almost steady, while there's more fluctuation on mobile devices. In October 2020, 25 desktop pages and 176 mobile pages make use of the API.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=644836316&format=interactive",
+  sheets_gid="1095648844",
+  sql_file="durable_storage_persist_usage.sql"
+  )
+}}
+
+The `persist()` API is less often called than the `estimate()` method. Only 176 mobile pages make use of this API, compared to 25 desktop websites. While usage on the desktop seems to remain at this low level, there is no clear trend on mobile devices.
```

Comment:
Yes, it would be cool if you could assist me here.

* I've checked [the sample URLs here](https://chromestatus.com/metrics/feature/timeline/popularity/1369). The significantly higher number of mobile sites could be due to the classified ad provider Locanto, which uses the API on different subdomains, all of which count as separate sites.
* Apart from that, using the persist() API on mobile devices probably makes more sense, as they tend to have a smaller storage capacity compared to desktop devices.